src/views/helpers/jobPrefIsPublic.js

```diff
@@ -1,3 +1,7 @@
 module.exports = (jobViewPref) => {
-  return jobViewPref === 'public';
-}
\ No newline at end of file
+  if (jobViewPref === 'public') {
```

Comment:
As discussed with @tdoran and @LawEKS not ideal, but ok for now. Added to refactor issue #59

Reply:
Due to the compare helper function, this could be deleted